<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlavorScreen</class>
 <widget class="QWidget" name="FlavorScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Chalkboard</family>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string>
  </property>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed{border-image:url(:/Common/Images/NextButtonPressed.png);}

QPushButton{border-image:url(:/Common/Images/NextButton.png);}

QPushButton{color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="SelectFlavorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Select a flavor</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="AdditionalChoiceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Additional choice</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="FlavorScrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="AdditionalOptionScrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="AdditionalScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>SelectFlavorLabel</zorder>
  <zorder>NextButton</zorder>
  <zorder>AdditionalChoiceLabel</zorder>
  <zorder>FlavorScrollArea</zorder>
  <zorder>AdditionalOptionScrollArea</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
